### DISCUSSION TITLE
Change footprint units to ug/m3

### DISCUSSION THREAD

**Discussion by inku0120** [uataq/stilt#128] [Users]
I have a query regarding replacing footprint units (ppm/umol/m2/s) with particle concentration (ug/m3). 
I want to use the concentrations  of a specie in (ug/m3) as a tracer concentration for the simulations. So that in the final result i would be able to visualize the particle concentration w.r.t to its trajectory.  just like in following figure. such that I can insert the concentration (ug/m3) [measured from instruments] at the receptor into the STILT.
![Screenshot (138)](https://user-images.githubusercontent.com/84829143/119985220-00af4c80-bfe0-11eb-9d4e-aae879d3ef75.png)

Is it feasible? What are the changes I need to make in the rstilt.r or which variable needs to be modified for this?

**Comment by benfasoli** [comment]
STILT uses theoretical "particles" to represent massless air parcels, which differs from physical particles or aerosols in the atmosphere. Since there is no mass associated with the particles, they aren't typically used to calculate a mass concentration (ug/m3).

It isn't really clear what you're trying to model. Are you using STILT forward or backward in time? If you're looking to model the mass concentration downstream from an emissions source, HYSPLIT may be a better fit for your use case than STILT.
